server/src/main/java/org/opensearch/common/Rounding.java

```diff
@@ -98,7 +98,7 @@ long roundFloor(long utcMillis) {
             }
 
             @Override
-            long extraLocalOffsetLookup() {
+            public long extraLocalOffsetLookup() {
```

Comment:
@jainankitk  What is the reason to change the visibility level? this is internal API and should stay as such

Reply:
The offset value is needed for creating the correct range buckets in `FilterRewriteHelper::createFilterForAggregations`. Some of the calendar intervals like month/quarter/year can have varying number of days, which is stored in extraLocalOffsetLookup. Do you see any harm in making just getter public? If yes, do you have a workaround in mind?

Reply:
@jainankitk please take a loot at https://github.com/opensearch-project/OpenSearch/pull/11392

Reply:
That's a good one, thanks!